Mondee Watch App/View/Play/GameOptionView.swift

```diff
@@ -67,6 +68,8 @@ struct GameOptionView: View {
                         gameState.successGameEarly()
                         gameStatus = .success
                         selection = .game
+                        gameState.isSuccessActive = true
+                        
```

Comment:
띄어쓰기 디테일 수정하시는것이 어떠신가요?
```suggestion
                        gameState.isSuccessActive = true
```